<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plot3DAreaDock</class>
 <widget class="QWidget" name="Plot3DAreaDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>782</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMain">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QLabel" name="lGeometry">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Geometry:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QDoubleSpinBox" name="sbWidth">
         <property name="suffix">
          <string> cm</string>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lBottomMargin">
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lTopMargin">
         <property name="text">
          <string>Left</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lLeftMargin">
         <property name="text">
          <string>Top</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QDoubleSpinBox" name="sbHeight">
         <property name="suffix">
          <string> cm</string>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="TimedLineEdit" name="leName"/>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer_10">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lRightMargin">
         <property name="text">
          <string>Height</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QCheckBox" name="chkVisible">
         <property name="text">
          <string>visible</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>58</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="2">
        <widget class="QDoubleSpinBox" name="sbLeft">
         <property name="prefix">
          <string/>
         </property>
         <property name="suffix">
          <string> cm</string>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lName">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lComment">
         <property name="text">
          <string>Comment</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QDoubleSpinBox" name="sbTop">
         <property name="suffix">
          <string> cm</string>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="ResizableTextEdit" name="teComment">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>src/kdefrontend/widgets/TimedLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>ResizableTextEdit</class>
   <extends>QTextEdit</extends>
   <header>src/kdefrontend/widgets/ResizableTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>leName</tabstop>
  <tabstop>teComment</tabstop>
  <tabstop>sbLeft</tabstop>
  <tabstop>sbTop</tabstop>
  <tabstop>sbWidth</tabstop>
  <tabstop>sbHeight</tabstop>
  <tabstop>chkVisible</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
